### PR TITLE
WIN32 - Copy dlls from external project to right place

### PR DESCRIPTION
**Pull request by juliencastelneau**
Creation of a new macro:
If a dll is generated in a plugin from an extenal project (MUSIC for instance), 
the dll poduced  is now copied to the right directory

### CODE REVIEW

File: packaging/windows/WindowsPackaging.cmake [hunk 1]
```diff
@@ -115,6 +115,7 @@ list(APPEND
   ${TTK_DIR}/bin/Release 
   ${dtk_DIR}/bin/Release 
   ${RPI_DIR}/bin/Release 
+  ${ZLIB_DIR}/bin
```
Comment (Florent2305) [inline packaging/windows/WindowsPackaging.cmake:118]: zlib ne concerne pas medInria

File: src/cmake/module/set_plugin_install_rules.cmake [hunk 2]
```diff
@@ -57,6 +57,35 @@ endmacro()
 
 ################################################################################
 ##############################   LEGACY   ######################################
+macro(set_plugin_install_rules_legacy_generic target dest)
+get_property(GENERATOR_MULTI_CONFIG GLOBAL PROPERTY GENERATOR_IS_MULTI_CONFIG)
```
Comment (Florent2305) [inline src/cmake/module/set_plugin_install_rules.cmake:61]: Super ;)

**Review by Florent2305 — CHANGES_REQUESTED** [review]
Très très bien juste la partie zlib à retirer

File: packaging/windows/WindowsPackaging.cmake [hunk 3]
```diff
@@ -124,11 +124,13 @@ file(GLOB_RECURSE vtk_files LIST_DIRECTORIES true \"${VTK_DIR}/bin/*.dll\")
 file(GLOB_RECURSE dtk_files LIST_DIRECTORIES true \"${dtk_DIR}/bin/*.dll\")
 file(GLOB_RECURSE dcm_files LIST_DIRECTORIES true \"${QtDCM_DIR}/bin/*.dll\")
 file(GLOB_RECURSE qt5_files LIST_DIRECTORIES true \"${QT_BINARY_DIR}/*.dll\")
+file(GLOB_RECURSE zlib_files LIST_DIRECTORIES true \"${ZLIB_DIR}/bin/*.dll\")
```
Comment (mathildemerle) [inline packaging/windows/WindowsPackaging.cmake:127]: still zlib changes in this file

**Review by Florent2305 — APPROVED** [review]

**Pull request merged**